<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifierVisiteur</class>
 <widget class="QDialog" name="ModifierVisiteur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="NomMod">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>60</y>
     <width>481</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ProvMod">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>116</y>
     <width>481</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>219</y>
     <width>291</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="ComfMod">
      <property name="text">
       <string>Comfirmer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AnnulModifVisiteur">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="AdresseMod">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>88</y>
     <width>481</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>88</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Modification </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>116</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Provenance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>27</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
